spec/controllers/static_pages_controller_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
```

Comment:
Same with this entire file. Didn't we already merge this feature?

Reply:
No. It is part of this  PR.